fusor-ember-cli/app/templates/add-node-registration.hbs

```diff
@@ -0,0 +1,74 @@
+{{#em-modal configName="bs" id="addNodeModal" closeIfClickedOutside=handleOutsideClick open-if=openAddNodeRegistrationModal close-if=closeAddNodeRegistrationModal}}
+  {{#em-modal-title}}
+    {{#em-modal-toggler class="close"}}
+      <span aria-hidden="true">×</span><span class="sr-only">Close</span>
+    {{/em-modal-toggler}}
+    <h4 class="modal-title">Add Node(s) to {{nodeInfo.address}}</h4>
+  {{/em-modal-title}}
+
+  {{#em-modal-body}}
+    <div class="row">
+      <div class="col-xs-12">
+        <p>Enter information for additional nodes that should be registered to {{nodeInfo.address}}</p>
+      </div>
+    </div>
+    <div class="row">
+      <div class="col-xs-offset-1 col-xs-11">
+        <form class="form-horizontal add-node-form">
+
+          {{#base-f cssId="addDriverInput"
+                    label="Driver"
+                    labelSize='col-xs-3'
+                    inputSize='col-xs-4'}}
+            <div class="add-node-uneditable-input-container"><span>{{nodeDriverHumanized}}</span></div>
+          {{/base-f}}
+
+          {{#base-f cssId="addAddressInput"
+                    label=newNodeAddressLabel
+                    labelSize='col-xs-3'
+                    inputSize='col-xs-4'}}
+            <div class="add-node-uneditable-input-container"><span>{{nodeInfo.address}}</span></div>
+          {{/base-f}}
+
+
+          {{#base-f cssId="addUsernameInput"
+                    label=newNodeUsernameLabel
+                    labelSize='col-xs-3'
+                    inputSize='col-xs-4'}}
+            <div class="add-node-uneditable-input-container"><span>{{nodeInfo.username}}</span></div>
+          {{/base-f}}
+
+          {{text-f  cssId="manualPasswordInput"
+                    label=newNodePasswordLabel
+                    type='password'
+                    labelSize='form-label-nowrap col-xs-3'
+                    inputSize='col-xs-4'
+                    value=nodeInfo.password
+                    isRequired=true}}
+
+          {{#each nodeInfo.macAddresses as |macAddress key| }}
+            {{new-node-registration-mac-address macAddress=macAddress index=key}}
```

Comment:
Actually, in this case it's intentionally not so they can blank one out if they made a mistake.  The submit only enables when all filled out fields are valid, though